.github/workflows/build_wheels_linux.yml

```diff
@@ -84,6 +85,7 @@ jobs:
       - name: Install torch dependency
         run: |
           source "${BUILD_ENV_FILE}"
+          set -ex
```

Comment:
Curious to know what difference does this make? thx!

Reply:
-x will print the full bash command that is run (good for debuggability, and ideally we should do so for every step in each workflow)
-e will make the script exit upon first failure.

Reply:
thanks for the info.